src/main/java/br/com/quintoandar/javajwt/QuintoAndarKeycloakJwtBean.java

```diff
@@ -0,0 +1,89 @@
+package br.com.quintoandar.javajwt;
+
+import org.jose4j.jwk.RsaJsonWebKey;
+import org.jose4j.jwt.consumer.InvalidJwtException;
+import org.jose4j.jwt.consumer.JwtConsumer;
+import org.jose4j.jwt.consumer.JwtConsumerBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.IOException;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.interfaces.RSAPublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+import java.util.Map;
+import java.util.Optional;
+
+public class QuintoAndarKeycloakJwtBean implements QuintoAndarKeycloakJwt {
```

Comment:
I think that the class must be annotated with `@Component` or other spring annotation for class